assets/src/blocks/TakeActionBoxout/TakeActionBoxoutFrontend.js

```diff
@@ -11,12 +11,7 @@ export const TakeActionBoxoutFrontend = ({
   imageAlt,
   className,
 }) => (
-  <section
-    className={`cover-card action-card ${className || ''}`}
-    style={{
-      backgroundImage: `linear-gradient(180deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0)), url(${imageUrl})`
-    }}
-  >
+  <section className={ `boxout ${ className || '' }` }>
     <a
       data-ga-category="Take Action Boxout"
       data-ga-action="Image"
```

Comment:
Maybe something for a follow-up ticket, but the analytics will need to be updated: here for instance they no longer make sense since the image is no longer full-width